src/lib/gundb/UserStorageClass.js

```diff
@@ -1886,36 +1877,23 @@ export class UserStorage {
     return `${type}${sufix}`
   }
 
-  async _extractProfileToShow(initiatorType, initiator, address): Gun {
-    const getProfile = async (group, value) => {
-      logger.debug('extractProfile:', { group, value })
-
-      // Need to verify if user deleted, otherwise the gun will stuck here and feed wont be displayed
-      // The group will contain null value in case user was deleted
-      const gunGroupIndexValue = this.gun.get(group).get(value)
-      const groupValue = await gunGroupIndexValue.then()
-      logger.debug('extractProfiler result :', { group, value, groupValue })
-
-      if (!isNull(groupValue) && Gun.node.is(groupValue)) {
-        return {
-          gunProfile: gunGroupIndexValue.get('profile'),
-        }
-      }
-      logger.warn('_extractProfileToShow invalid profile', { group, value, groupValue })
-    }
+  _getProfileNode(initiatorType, initiator, address): Gun {
+    const getProfile = (group, value) =>
```

Comment:
it's seems it's a reason why tests are failing
@yaroslav-fedyshyn-nordwhale pls look at this ASAP

Reply:
Verification was added to detect whether the account was deleted and its no longer needed as you commented on the lines where indexed fields become null on account delete action - so there won't be a situation when the index will contain null. (https://github.com/GoodDollar/GoodDAPP/blob/master/src/lib/gundb/UserStorageClass.js#L1492).
And I returned to the previous implementation without verifying the indexed field is null.

The bug happens because of 'Gun.node.is(groupValue)' returns false and 'profileNode' becomes undefined - so there was no full name displayed on feed card.